_posts/2022-04-26-1.15-announcement.md

```diff
@@ -0,0 +1,434 @@
+1.15, we have introduced a new execution mode named 'thread' mode, for which Python 
+user-defined functions will be executed in the JVM as a thread instead of a separate 
+Python process. Benchmarks have shown that throughput could be increased by 2x in 
+common scenarios such as JSON processing. Processing latency is also decreased from 
+several seconds to micro-seconds. It should be noted that since this is still the first 
+release of 'thread' mode, it currently only supports Python ScalarFunction which is used 
+in Python Table API & SQL. We're planning to extend it to other areas in which Python 
+user-defined functions could be used in the next releases.
+
+
+### Other 
+
+Further work has been done on the 
+[connector testing framework](https://github.com/PatrickRen/flink/tree/master/flink-test-utils-parent/flink-connector-testing). If you want to contribute a connector or improve on one, you should definitely have a 
+look. 
+
+Some long-awaited features have been added, including the 
+[CSV format](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/formats/csv/) 
+and the [small file compaction](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/filesystem/#compaction) 
+in the unified sink interface. 
+
+The sink API has been upgraded 
+to [version 2](https://github.com/apache/flink/blob/master/flink-core/src/main/java/org/apache/flink/api/connector/sink2/StatefulSink.java)
+and we encourage every connector maintainer to upgrade to this version.
+
+
+## Summary
+
+Apache Flink is now easier to operate, made even more progress towards aligning stream and 
+batch processing, became more accessible through improvements in the SQL components, 
+and now integrates better with other technologies. 
+
+It is also worth mentioning that 
+the community has set up a new home for 
+the [CDC connectors](https://ververica.github.io/flink-cdc-connectors/release-2.1/index.html), 
+the [connector repository](https://cwiki.apache.org/confluence/display/FLINK/Connectors) 
+will be externalized 
+([with the Elasticsearch sink as a first example](https://github.com/apache/flink-connector-elasticsearch/)), 
+and there is now a 
+[Kubernetes operator](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/) 
+([announcement blogpost](https://flink.apache.org/news/2022/04/03/release-kubernetes-operator-0.1.0.html) 
+maintained by the community.
+
+Moving forward, the community will continue to focus on making Apache Flink a true 
+unified stream and batch processor and work on better integrating Flink into the cloud-native 
+ecosystem. 
+
+
+## Upgrade Notes
+
+While we aim to make upgrades as smooth as possible, some of the changes require users 
+to adjust some parts of the program when upgrading to Apache Flink 1.15. Please take a 
+look at the
+[release notes](https://nightlies.apache.org/flink/flink-docs-release-1.14/release-notes/flink-1.14/)
```

Comment:
@joemoe  Here should be `https://nightlies.apache.org/flink/flink-docs-release-1.15/release-notes/flink-1.15/`